assets/i18n/en_US.json

```diff
@@ -136,7 +137,10 @@
         "unsupportedPatchVersion": "Patch is not supported for this app version. Enable the experimental toggle in settings to proceed.",
 
         "newPatchDialogText": "This is a new patch that has been added since the last time you have patched this app.",
-        "newPatch": "New patch"
+        "newPatch": "New patch",
+
+        "patchesChangeWarningDialogText": "It is recommended to use the default selection of patches because changing it may cause unexpected issues.\n\nIf you know what you are doing, you can enable \"Allow changing selection\" in the settings.",
```

Comment:
```suggestion
        "patchesChangeWarningDialogText": "It is recommended to use the default selection of patches because changing it may cause unexpected issues.\n\nIf you know what you are doing, you can enable \"Enable changing selection\" in the settings.",
```